scripts/lambda/README.md

```diff
@@ -0,0 +1,6 @@
+# Lambda Scripts
```

Comment:
We could probably just put it in a stand alone repository.  That will make it easier to setup CI/CD too.

Reply:
That makes sense - the environment is independent of the rest of the project, anyway. I'll close this after reviews and make `avrae/lambda-scripts` instead.